parser/bench_test.go

```diff
@@ -17,34 +17,103 @@ import (
 	"testing"
 )
 
-func BenchmarkSysbenchSelect(b *testing.B) {
+func sysebchSelect(parser *Parser, sql string, num int, b *testing.B) {
```

Comment:
This function has nothing to do with `sysbench`.

Reply:
As you can see, old function name declared as `SysbenchSelect`. I just adopted same name.

Reply:
It's an extracted function, it run parsing for N times and doesn't have to be called by BenchmarkSysbenchSelect.